R/mread.R

```diff
@@ -600,6 +600,11 @@ mread_cache <- function(model = NULL,
                         preclean = FALSE, 
                         capture = NULL, ...) {
   
+  if (charthere(model, "/")) {
```

Comment:
I'm not sure where mrgsolve stands with Windows compatibility, but this would fail if someone on Windows passed "\\", right?  

As an alternative check/condition, the first thing that comes to mind is

```
!identical(basename(model), as.character(model))
```

(with `as.character` there in case someone passes in an `fs_path` object)